core/src/main/java/org/mapstruct/BeanMapping.java

```diff
@@ -14,10 +14,51 @@
 import static org.mapstruct.NullValueCheckStrategy.ON_IMPLICIT_CONVERSION;
 
 /**
+ * // TODO should I give some nullValueMappingStrategy example or not? <br>
+ * // TODO should I give some example with qualifiedBy or it's better to do it in the annotation @Qualifier?<br>
```

Comment:
I guess in general: if we have a type to stick javadoc on: do it in the type.. So here in @Qualifier. Otherwise here.